src/create-context.js

```diff
@@ -22,17 +22,20 @@ export function createContext(defaultValue) {
 				this.shouldComponentUpdate = _props => {
 					if (this.props.value !== _props.value) {
 						subs.some(c => {
-							c.context = _props.value;
-							enqueueRender(c);
+							c[0].context = _props.value;
+							if (!c[1] || c[1](_props.value, this.props.value)) {
+								enqueueRender(c[0]);
+							}
 						});
 					}
 				};
 
-				this.sub = c => {
-					subs.push(c);
+				this.sub = (c, shouldUpdate) => {
+					const entry = [c, shouldUpdate];
```

Comment:
I think `const entry = { _component: c, _shouldUpdate: shouldUpdate };` ends up being smaller?

Reply:
This adds 1 byte